productcat/src/main/java/com/visenze/productcat/android/ProductCat.java

```diff
@@ -108,13 +125,27 @@ public void cancelSearch() {
     }
 
     public void imageSearchResultPage(final ImageSearchParams params) {
```

Comment:
can you do the same for the imageSearch method? we want to standardize.

Reply:
done